scripts/logger.py

```diff
@@ -24,7 +24,7 @@
 class Logger(metaclass=Singleton):
     def __init__(self):
         # create log directory if it doesn't exist
-        log_dir = os.path.join('..', 'logs')
+        log_dir = os.path.join('.', 'logs')
```

Comment:
Isn't this gona create scripts/logs/ ?
Where's `'.'`? Is that GUARANTEED to be scripts/ seeing as logger.py is in scripts/

Reply:
I see this relative-file-location issue cropping up in a few places. Maybe we need something like 
```
script_folder = os.path.dirname(os.path.abspath(__file__))
proj_root_folder = os.path.dirname(script_folder)
```

... but if we need this in multiple places, better put some `getProjectRoot` func in utils.py or something